<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportOVAccDlg</class>
 <widget class="QDialog" name="ImportOVAccDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>当前账户</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QLineEdit" name="edtCurAcc">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>当前帐套</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtCurSuite">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>最后月份</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="edtLastMonth">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>最大凭证号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="edtMaxPzNum">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnFile">
       <property name="text">
        <string>老版账户文件</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QLineEdit" name="edtFileName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>导入凭证集年月</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="edtDate">
       <property name="displayFormat">
        <string>yyyy-M</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>开始凭证号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spnStart"/>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>结束凭证号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QSpinBox" name="spnEnd"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ImportOVAccDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
